.github/resources/scripts/user_manager.py

```
@@ -0,0 +1,375 @@
+        print("\n" + "="*80)
+        print("📋 KUBECONFIG CREATION INSTRUCTIONS")
+        print("="*80)
+
+        for namespace, users in self.workspace_users.items():
+            print(f"\n🏗️  WORKSPACE: {namespace.upper()}")
+            print("-" * 50)
+
+            for username, permission_type in users.items():
+                perm_desc = self.permission_types[permission_type]["description"]
+                print(f"\n🔑 {username.upper()} ({perm_desc}):")
+                print(f"   # Get the token")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)